lib/Espfc/src/Target/TargetESP32c3.h

```diff
@@ -0,0 +1,156 @@
+#pragma once
+
+#include "Esp.h"
+#include "Debug_Espfc.h"
+
+#define ESPFC_INPUT
+#define ESPFC_INPUT_PIN 3 // ppm
+
+#define ESPFC_OUTPUT_COUNT 4
+#define ESPFC_OUTPUT_0 2
+#define ESPFC_OUTPUT_1 4
+#define ESPFC_OUTPUT_2 18
+#define ESPFC_OUTPUT_3 19
+
+#define ESPFC_SERIAL_0
+#define ESPFC_SERIAL_0_DEV Serial
+#define ESPFC_SERIAL_0_DEV_T HardwareSerial
+#define ESPFC_SERIAL_0_TX 21
+#define ESPFC_SERIAL_0_RX 20
+#define ESPFC_SERIAL_0_FN (SERIAL_FUNCTION_MSP)
+#define ESPFC_SERIAL_0_BAUD (SERIAL_SPEED_115200)
+#define ESPFC_SERIAL_0_BBAUD (SERIAL_SPEED_NONE)
+
+#define ESPFC_SERIAL_REMAP_PINS
+#define ESPFC_SERIAL_DEBUG_PORT SERIAL_UART_0
+#define SERIAL_TX_FIFO_SIZE 0xFF
+
+#define ESPFC_SPI_0
+#define ESPFC_SPI_0_DEV SPI1
+#define ESPFC_SPI_0_SCK 6
+#define ESPFC_SPI_0_MOSI 5
+#define ESPFC_SPI_0_MISO 7
+
+#define ESPFC_SPI_CS_GYRO 10
+#define ESPFC_SPI_CS_BARO 12
+
+#define ESPFC_I2C_0
+#define ESPFC_I2C_0_SCL 9
+#define ESPFC_I2C_0_SDA 8
+//#define ESPFC_I2C_0_SOFT
+
+#define ESPFC_BUZZER
+#define ESPFC_BUZZER_PIN 0
+
+#define ESPFC_ADC_0
+#define ESPFC_ADC_0_PIN 1
+
```

Comment:
Two 12-bit SAR ADCs, at most 6 channels

Reply:
we can also add `ESPFC_ADC_1` and `ESPFC_ADC_1_PIN` and set pins to `-1` initially.